docs/examples.md

```diff
@@ -372,6 +372,20 @@ systemd:
         [Install]
         WantedBy=multi-user.target
 ```
+## GRUB password
+
+This example adds a superuser to GRUB and sets a password. Users without the given username
+and password will not be able to access GRUB command-line, modify kernel command-line arguments, or boot non-default OSTree deployments.
```

Comment:
The first instance of `command-line` is a compound noun and should be `command line`.